control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -116,6 +119,9 @@ func (r *HostedControlPlaneReconciler) SetupWithManager(mgr ctrl.Manager) error
 		return fmt.Errorf("failed setting up with a controller manager %w", err)
 	}
 
+	// Set based on SCC capability
```

Comment:
can we may be provide some "why" context here? e.g "SCC automatically provide a Security Context and user within an automatically generated UID range. When SCC are not available in the management cluster we want to explicitly set a non root security context."

Reply:
Updated.